### PR TITLE
improve initial image zoom and centering

### PR DESCRIPTION
**Pull request by eliotjordan**
Starting off with a Happy New Year pull request. :sparkles: This address #2. Much of the logic borrowed from [turban/Leaflet.Zoomify](https://github.com/turban/Leaflet.Zoomify).

### CODE REVIEW

**Comment by mejackreed** [comment]
This looks great. I am running into a bug though I am trying to work through, maybe you can help? 

First rebase master onto the branch (just added new layer switcher).

Switch layer to "Princeton" then to "Klokan" and then back to "Princeton". The tiles never get loaded for me. I am trying to work through what is going on here.

**Comment by mejackreed** [comment]
@eliotjordan I think it has something to do with a layer being added at a zoom level it does not support.

File: leaflet-iiif.js [hunk 1]
```diff
@@ -44,8 +44,14 @@ L.TileLayer.Iiif = L.TileLayer.extend({
...
-      // Try to center the map a bit
-      map.setView([-_this.options.tileSize / 2, _this.options.tileSize / 4], 1);
+      // Find best zoom level and center map
+      var initialZoom = _this._getInitialZoom(map.getSize()),
+        imageSize = _this._imageSizes[initialZoom],
+        sw = map.options.crs.pointToLatLng(L.point(0, imageSize.y), initialZoom),
+        ne = map.options.crs.pointToLatLng(L.point(imageSize.x, 0), initialZoom),
+        bounds = L.latLngBounds(sw, ne);
+
+      map.fitBounds(bounds);
```
Comment (mejackreed) [inline leaflet-iiif.js]: If you add `true` as the second parameter it should fix the issue I was seeing. 

http://leafletjs.com/reference.html#map-fitboundsoptions 

It should reset the map view.

**Comment by eliotjordan** [comment]
Yep. That worked. Just pushed the commit.

**Comment by mejackreed** [comment]
:+1: awesome thanks

**Pull request merged**